sunpy/visualization/tests/test_mapcubeanimator.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+import os
+
+import matplotlib.pyplot as plt
+import pytest
+
+import sunpy.map
+from sunpy.visualization import mapcubeanimator
+from sunpy.data import test
+
+
+@pytest.fixture
+def map_animator():
+    testpath = test.rootdir
+    aia_file = os.path.join(testpath, "aia_171_level1.fits")
+    mapcube = sunpy.map.Map(aia_file)
+    cube = sunpy.map.MapCube(mapcube)
+    return mapcubeanimator.MapCubeAnimator(cube)
+
+
+# test for mapcubeanimator instance
+def test_mapcubeanimator_instances(map_animator):
+
+    assert isinstance(map_animator, mapcubeanimator.MapCubeAnimator)
+    assert map_animator.interval is 200
+    assert map_animator.annotate is True
+    assert map_animator.slider_ranges == [[0, 1]]
+
+    assert map_animator.timer is None
+
+    assert map_animator.active_slider == 0
+    assert map_animator.button_func == []
+    assert map_animator.remove_obj == []
+
+
+def test_updatefig(map_animator):
+    t = map_animator.updatefig(0, map_animator.im, 10)
+    assert t is None
+
+
+def test_plot_start_image(map_animator):
+    t = map_animator.plot_start_image(plt.axes([0, 0, 0, 0]))
+    s = map_animator.mapcube[0].plot(axes=plt.axes([0, 0, 0, 0]))
+    assert t.axes == s.axes
```

Comment:
This test as it stands isn't useful. Can you create a figure test for this?

Reply:
@nabobalis 
I can use generic map instead of mapcube, but I am not sure about mapcuabeanimator  of figures,It is possible for imageanimator, but there is not a test for it in that module also

Reply:
I am not sure I follow. The figure test would be that you create the mapcube_animator, it would look correct to the inputs you gave it at the start. You hash the figure and add it figure hashes. 

The figure test isn't the direct comparison of two figures like a normal test is.

Reply:
Although, I have never actually created one myself.